ngraph/python/requirements.txt

```diff
@@ -1,2 +1,5 @@
+Cython==0.29.21
 numpy
+pip==20.2.2
```

Comment:
We don't need to include `pip` or `setuptools` to requirements.

Reply:
Should be in ngraph/python/requirements-dev.txt